app/src/main/java/com/iron/espresso/domain/repo/StudyRepository.kt

```diff
@@ -0,0 +1,8 @@
+package com.iron.espresso.domain.repo
+
+import com.iron.espresso.model.response.study.StudyListResponse
+import io.reactivex.Single
+
+interface StudyRepository {
+    fun getStudyPagingList(studyIds: String): Single<StudyListResponse>
```

Comment:
```suggestion
    fun getStudyPagingList(studyIds: List<String>): Single<StudyListResponse>
```
이렇게 하고 RemoteDataSource에서 joinToString 하도록 변경하는게 좋지 않을까 싶음

Reply:
@wswon 수정했습니다